src/analyzer/qa/private_vars_leading_underscore.rs

```diff
@@ -0,0 +1,142 @@
+use std::collections::HashSet;
+
+use solang_parser::pt::{self, FunctionTy, Loc};
+use solang_parser::{self, pt::SourceUnit};
+
+use crate::analyzer::ast::{self, Target};
+use crate::analyzer::utils::get_32_byte_storage_variables;
+
+pub fn private_vars_leading_underscore(source_unit: SourceUnit) -> HashSet<Loc> {
+    //Create a new hashset that stores the location of each qa target identified
+    let mut qa_locations: HashSet<Loc> = HashSet::new();
+
+    extract_matches_in_variables(&source_unit, &mut qa_locations);
+
+    extract_matches_in_functions(source_unit, &mut qa_locations);
```

Comment:
Can we separate `extract_matches_in_variables` and `extract_matches_in_functions` into two separate QA mods so that someone could use them independently from the other.

Reply:
Yes of course. I'll change it tomorrow 🤜🤛

Reply:
Fixed